ember-toucan-core/src/components/form/radio-field.hbs

```diff
@@ -0,0 +1,42 @@
+<div
+  class="flex flex-col"
+  data-root-field={{if @rootTestSelector @rootTestSelector}}
+>
+  <Form::Field as |field|>
+    <field.Control class="rounded-sm" data-control>
+
+      <label class="flex flex-row space-x-3" for={{field.id}}>
+        <Form::Controls::Radio
+          aria-invalid={{if @error "true"}}
+          aria-describedby="{{if @error field.errorId}}"
+          id={{field.id}}
+          @isDisabled={{@isDisabled}}
+          @isChecked={{@isChecked}}
+          @value={{@value}}
+          @onChange={{@onChange}}
+          ...attributes
+        />
+
+        <div class="flex flex-col">
+          <span
+            class="type-md-tight block leading-4
+              {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
+            data-label
+          >{{@label}}</span>
+
+          {{#if @hint}}
+            <field.Hint class="mt-0.5" data-hint>{{@hint}}</field.Hint>
+          {{/if}}
+        </div>
+      </label>
+    </field.Control>
+
+    {{#if @error}}
+      <field.Error
+        class="mt-0.5"
+        id={{field.errorId}}
+        data-error
+      >{{@error}}</field.Error>
+    {{/if}}
```

Comment:
I don't think anyone will actually use this individual radio error over what we're planning to provide via the `Fieldset`-wrapped option, but I _suppose_ they could.  We're still ironing things out from a design perspective here, so we may remove this after further design review, or it may stay!  Here's what the Fieldset option may look like:

```hbs
<Form::Fieldset
  @label='Label'
  @hint='Extra information about the fieldset'
  @error='Error message'
>
    <Form::RadioField @value="option1" @label='Label1' name="label1" />
    <Form::RadioField @value="option2" @label='Label2' name="label2" />
    <Form::RadioField @value="option3" @label='Label3' name="label3" />
</Form::Fieldset>
```

And then we get the error border/shadow for free as seen at: https://25c29909.ember-toucan-core.pages.dev/docs/components/fieldset#all-ui-states